utils/exportHuds/exportHudsController.ts

```diff
@@ -0,0 +1,91 @@
+
+export async function createFile(idExportHuds) {
+    return new Promise(async (resolve, reject) => {
+
+        const peticionExport: any = await ExportHudsModel.findById(idExportHuds);
+        let fechaCondicion = null;
+        let query = {
+            'paciente.id': peticionExport.pacienteId,
+            'estadoActual.tipo': 'validada'
+        };
+        if (peticionExport.fechaDesde && peticionExport.fechaHasta) {
+            fechaCondicion = {
+                $gte: moment(peticionExport.fechaDesde).startOf('day').toDate(),
+                $lte: moment(peticionExport.fechaHasta).endOf('day').toDate()
+            };
+            query['ejecucion.fecha'] = fechaCondicion;
+        }
+        if (peticionExport.tipoPrestacion) {
+            query['solicitud.tipoPrestacion.conceptId'] = peticionExport.tipoPrestacion;
+        }
+        const prestaciones: any = await Prestacion.find(query);
+        const fecha = moment(peticionExport.createAt).format('YYYY-MM-DD');
+
+        const archive = archiver('zip', {
+            zlib: { level: 9 }
+        });
+        const metadata = {
+            user: peticionExport.user
+        };
+        const options = ({
+            filename: `HUDS-${peticionExport.pacienteNombre}-${fecha}`,
+            contentType: 'application/zip',
+            metadata
+        });
+        const HudsFiles = getHUDSExportarModel();
+        try {
+            HudsFiles.writeFile(
+                options
+                ,
+                archive,
+                // tslint:disable-next-line: no-shadowed-variable
+                (_error: any, archive) => {
+                    if (_error) {
+                        return reject();
+                    }
+                    peticionExport.idHudsFiles = archive._id;
+                    peticionExport.status = 'completed';
+                    peticionExport.updatedAt = new Date();
+                    peticionExport.save();
+                    return resolve();
+                }
+            );
+            const objectLog = {
+                usuario: peticionExport.user.usuario,
+                huds: options.filename,
+                organizacion: peticionExport.user.organizacion
+            };
+            exportHudsLog.info('exportaHuds', objectLog);
+        } catch (error) {
+            throw error;
+        }
+        archive.on('error', (err) => {
+            throw err;
+        });
+        const getData = () => {
+            return Promise.all(prestaciones.map(async (prestacion: any) => {
+                let informe = new InformeRUP(prestacion.id, null, peticionExport.user);
+                let archivo = await informe.informe();
+                const nombreArchivo = prestacion.solicitud.tipoPrestacion.term;
+                const fechaArchivo = moment(prestacion.solicitud.fecha).format('YYYY-MM-DD');
+                archive.file(`${archivo}`, { name: `${nombreArchivo} - ${fechaArchivo}.pdf` });
```

Comment:
Cambiaría el orden para que queda ordenado por fecha. 

```suggestion
                archive.file(`${archivo}`, { name: `${fechaArchivo} - ${nombreArchivo}.pdf` });
```